### ISSUE TITLE
No Internet Access w/latest version

### ISSUE THREAD

**Issue by lacojim** [EliasKotlyar/Xiaomi-Dafang-Hacks#1728]
I am running this on three Wyze cameras. Two work perfectly. On refuses to connect to the internet. Of course because of this the time/date is never correct. etc/resolve.conf is correct, the same for all three. nslookup works on inhouse stuff, but fails on anything outside. I can ping the gateway just fine. When I ping an outside address, it DOES resolve the IP, just can't get there. I am at wits end. Spent about 10 hours trying to troubleshoot this today to no avail. Any ideas?

**Comment by jmtatsch** [comment]
Maybe you have duplicate 
MAC addresses among your cameras?

**Comment by lacojim** [comment]
Definitely not the case, but your comment did make me check it. I am hard assigning the dhcp IP via dnsmasq in my dd-wrt router. But that is no different than the other two.

**Comment by jmtatsch** [comment]
Did you try a fresh checkout and fresh config?

**Comment by lacojim** [comment]
No, but I found the issue. I had inadvertently blocked the ip range that camera was part of, in an attempt to block TPLink switches from calling home. The TPLink switches are on 215-229 IPs. I misstyped one of them as .121 which is where this particular camera was located. All fixed now. 

Thank you for your comments which made me look much closer at my router config.

I have so many devices, it is easy to over look. I need to add some more subnets I believe.
